README.md

```diff
@@ -945,6 +945,9 @@ Information about specific usages of PMEM-CSI are described in separate document
 
 Report a bug by [filing a new issue](https://github.com/intel/pmem-csi/issues).
 
+Before making your first contribution, be sure to read the [development documentation](DEVELOPMENT.md)
+for guidance on code quality and branches.
```

Comment:
Speaking of contributors (which we haven't had, so the question hasn't really come up): we currently don't use Signed-off-by because we haven't defined anywhere what it would mean.

I've always wondered whether we should write something about that...

Having said that, the commits are fine also with that statement. Please don't bother removing them.

Reply:
Sure. At least you aware you have no documented flow :-)
I'm somewhat biased, but as a reference, here is what [kata says about it](https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#certificate-of-origin). Pretty simple - just ascertaining the chain of trust on the code in case there is any future query.

Reply:
@grahamwhaley: Do you have automated PR checks that ensure that the signed-off-by is present in the commit?

Reply:
We do (for kata). Currently it is a check done by the [Zuul](https://github.com/kata-containers/community/wiki/The-SoB-check) CI, but previously we were using [pullapprove](https://github.com/kata-containers/community/wiki/PullApproveAlternatives) until they changed their licensing policy. If you check that link, there is a brief analysis of tools we considered using, including the kata homebrew [checkcommits](https://github.com/kata-containers/tests/tree/master/cmd/checkcommits) tool, that can perform a number of checks.
If you want any input or help with any of those, just shout.